components/intro/IntroMilestone.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div>
+        <p class="achievement">{{ $t(achievement) }}</p>
+        <p class="number">{{ $t(achievementNumber) }}</p>
+        <p class="achievement text-2xl mb-2">
```

Comment:
nit: BEM (checkout [this post](https://css-tricks.com/bem-101/)) is recommended here. For example,
```suggestion
        <p class="milestone__achievement">{{ $t(achievement) }}</p>
        <p class="milestone__number">{{ $t(achievementNumber) }}</p>
        <p class="milestone__unit text-2xl mb-2">
```

Reply:
Thanks for the suggestion, I fixed the class names and refactored components to make HTML which is in the file `intro.vue` look more structured.

Reply:
(6c0cc63)[https://github.com/pycontw/pycontw-2021/pull/60/commits/6c0cc637baf82e12d72d28f9f8bcdaf30a46c481]